stack_test.go

```diff
@@ -14,7 +14,7 @@ import (
 func TestNiceStack(t *testing.T) {
 	// Expected is actually "fixed up" to remove line
 	// numbers and local paths.
-	expected := `/usr/local/go/src/runtime/debug/stack.go <-- github.com/lytics/grid/stack_test.go <-- /usr/local/go/src/runtime/panic.go <-- github.com/lytics/grid/stack_test.go <-- github.com/lytics/grid/stack_test.go <-- /usr/local/go/src/testing/testing.go <-- /usr/local/go/src/testing/testing.go`
+	expected := `stack.go <-- stack_test.go <-- panic.go <-- stack_test.go <-- stack_test.go <-- testing.go <-- testing.go`
```

Comment:
Go modules means that the code is not at a predictable path. Instead of trying to identify where the line is between our dirs and parent dirs, just strip off everything dir-related. It'll still validate the ordering of things is correct.